### PR TITLE
Add the page with details.

### PR DESCRIPTION
**Pull request by ShukujiNeel13**
[Add page describing multi-tenant db under Matrix Event Bus](https://github.com/smartclean/smartclean.github.io/commit/2f68c9380b3acf67de675309b13e700c607048c9)

Note: This PR includes updates from a previous pull request: #72

### CODE REVIEW

File: docs/multi-tenant-db.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+### What is a shared database model
+
+1. This means the same database instance is shared, allowing the following two types of commits:
+   1. You can query data from it. 
+   2. Our system can write data to it. 
+2. Any standard [PostgreSQL](https://www.postgresql.org) database will work. 
+3. The database we use for this is called [TimeScale DB](https://docs.timescale.com) which is provided as an 
+extension to PostgreSQL. 
+4. A multi-tenant component means the component is shared (physically the same, with logical separation)
```
Comment (hello-error) [inline docs/multi-tenant-db.md:17]: A multi-tenant component means the data from projects owned by you is shared with some logical separation and query patterns.

File: docs/multi-tenant-db.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+### Means of data integration
+Allow you to query desired data pushed to a database by our platform, such as sensor data and Matrix events. 
+
+1. You may subscribe for this service in order to connect to this database.
```
Comment (hello-error) [inline docs/multi-tenant-db.md:23]: You may reach out to your account executive to setup the solution for you.

File: docs/multi-tenant-db.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+2. You may alternatively, create your own SQL database and give us access to write to it.
```
Comment (hello-error) [inline docs/multi-tenant-db.md:24]: You may alternatively, provision your own Postgres database instance with the TimeScale extension enabled and give us access to write to it.

File: docs/multi-tenant-db.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+Below diagram shows how SmartClean Matrix and subscribers of this service can access this database.
```
Comment (hello-error) [inline docs/multi-tenant-db.md:26]: The diagram below

File: docs/multi-tenant-db.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+### Connecting to the database
+1. An authorised connection to this PostgreSQL database is all that is required to start using it.
+2. Authorised means that a username and password will be required, since the instance must be secure.
+3. For connecting to a PostgreSQL database the following details are required:
+   1. **host** - A web URL identifying the database server.
+   2. **database** - Name given to the desired database on the server.
+   3. **user** - Name of user having access to the database.
+   4. **password** - Password for this user. 
+Note: These details are required whether the database is provided by us or provided by you.
```
Comment (hello-error) [inline docs/multi-tenant-db.md:57]: _The note can be deleted as its not necessary._

File: docs/multi-tenant-db.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+4. Below are some options for connecting to the database:
+   1. Use the PostgreSQL command line interface.
```
Comment (hello-error) [inline docs/multi-tenant-db.md:59]: 1. Use the PostgreSQL command line interface (psql).

File: docs/multi-tenant-db.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+   1. **host** - A web URL identifying the database server.
+   2. **database** - Name given to the desired database on the server.
+   3. **user** - Name of user having access to the database.
+   4. **password** - Password for this user. 
+Note: These details are required whether the database is provided by us or provided by you.
+4. Below are some options for connecting to the database:
+   1. Use the PostgreSQL command line interface. 
+   2. Use a library available in the programming language of your choice. 
+   3. Connect to it from a cloud platform or service of your choice. 
+   4. Majority of BI (business intelligence) service providers also allow you to connect to any PostgreSQL databases.
```
Comment (hello-error) [inline docs/multi-tenant-db.md:62]: 4. Majority of BI (business intelligence) service providers allow connection to PostgreSQL databases.

**Review by hello-error — CHANGES_REQUESTED** [review]

File: docs/multi-tenant-db.md (hunk 1 above)
Comment (ShukujiNeel13) [reply docs/multi-tenant-db.md]: Resolved by commit: https://github.com/smartclean/smartclean.github.io/pull/74/commits/f2bb068d9369ffd245dabfd277b712af828447ad

File: docs/multi-tenant-db.md (hunk 2 above)
Comment (ShukujiNeel13) [reply docs/multi-tenant-db.md]: Resolved by commit:
https://github.com/smartclean/smartclean.github.io/pull/74/commits/f2bb068d9369ffd245dabfd277b712af828447ad

File: docs/multi-tenant-db.md (hunk 3 above)
Comment (ShukujiNeel13) [reply docs/multi-tenant-db.md]: Resolved by commit:
https://github.com/smartclean/smartclean.github.io/pull/74/commits/f2bb068d9369ffd245dabfd277b712af828447ad

File: docs/multi-tenant-db.md (hunk 5 above)
Comment (ShukujiNeel13) [reply docs/multi-tenant-db.md]: Resolved by commit:
https://github.com/smartclean/smartclean.github.io/pull/74/commits/f2bb068d9369ffd245dabfd277b712af828447ad

File: docs/multi-tenant-db.md (hunk 4 above)
Comment (ShukujiNeel13) [reply docs/multi-tenant-db.md]: Resolved by commit:
https://github.com/smartclean/smartclean.github.io/pull/74/commits/f2bb068d9369ffd245dabfd277b712af828447ad

File: docs/multi-tenant-db.md (hunk 6 above)
Comment (ShukujiNeel13) [reply docs/multi-tenant-db.md]: Resolved by commit:
https://github.com/smartclean/smartclean.github.io/pull/74/commits/f2bb068d9369ffd245dabfd277b712af828447ad

File: docs/multi-tenant-db.md (hunk 7 above)
Comment (ShukujiNeel13) [reply docs/multi-tenant-db.md]: Resolved by commit:
https://github.com/smartclean/smartclean.github.io/pull/74/commits/f2bb068d9369ffd245dabfd277b712af828447ad

**Pull request merged**